contrib/node/src/python/pants/contrib/node/tasks/javascript_style.py

```diff
@@ -44,6 +43,13 @@ def register_options(cls, register):
             "--fail-slow", type=bool, help="Check all targets and present the full list of errors."
         )
         register("--color", type=bool, default=True, help="Enable or disable color.")
+        register(
+            "--file-extensions",
+            advanced=True,
+            type=list,
+            default=cls._DEFAULT_JS_EXTENSIONS,
+            help="File extensions that should be linted as JS",
```

Comment:
Nit: missing a period.

```suggestion
            help="File extensions that should be linted as JS.",
```